jetstream.go

```diff
@@ -100,19 +104,21 @@ func (m *Msg) parseJSMsgMetadata() (*JetStreamMsgMetaData, error) {
 
 const jsStreamUnspecified = "not.set"
 
-type jsAcKOpts struct {
+type jsAckOpts struct {
```

Comment:
This need to be a struct? Just embed the string into jsOpts?

Reply:
doesnt need to be now no, used to have a few more ack related things there